libgearman/protocol/submit.h

```diff
@@ -48,13 +48,6 @@ gearman_return_t submit(gearman_universal_st&,
                         const gearman_string_t &function,
                         const gearman_string_t &workload);
 
-gearman_return_t submit_background(gearman_universal_st&,
```

Comment:
No objection on my part.
It would be helpful you express your concerns to remaining [points](https://github.com/gearman/gearmand/issues/72#issuecomment-303728212)